### ISSUE TITLE
SEE-OTHER and its relationship to Bound-withs

### ISSUE THREAD

**Issue by marlo-longley** [sul-dlss/SearchWorks#3128]
This came up in my investigation of #3099 

One of the UX gaols that came up when talking with @saseestone was to change the UX of this yellow highlighted area that makes a user click up to a parent item in order to make a request. 

You can see this flow in action here: https://searchworks.stanford.edu/view/8594988.
<img width="1435" alt="Screenshot 2023-05-03 at 14 10 44" src="https://user-images.githubusercontent.com/1328900/236050747-aa150c3a-fd3a-464b-accf-fa83eb10c8ac.png">

However, the majority of Bound-with titles actually do not follow this flow. They have a request button directly on their own SearchWorks page. For example: https://searchworks.stanford.edu/view/2320068. Here 061 .T355 no. 12 has its own Request button, which leads to the following requests link: https://requests.stanford.edu/requests/new?item_id=2320068&origin=SAL&origin_location=SOUTH-MEZZ

<img width="1473" alt="Screenshot 2023-05-03 at 14 13 06" src="https://user-images.githubusercontent.com/1328900/236051124-50f50562-8d0b-4949-8b77-32383bae1bad.png">

The difference between these two displays is the presence of SEE-OTHER in the record 999 notes. Both are BW-CHILD.
In the first record above, the 999 field reads:
```     
a| 488.21 .O98 1906 w| DEWEY c| 1 i| 8594988-2001 l| SEE-OTHER m| SAL3 r| Y s| Y t| STKS-MONO u| 6/1/2010 x| BW-CHILD
```
While the second record's 999 reads:
```
a| 061 .T355 NO.12 w| DEWEY c| 1 i| 001ALQ2739 d| 8/7/1996 l| SOUTH-MEZZ m| SAL r| Y s| Y t| STKS-MONO u| 9/10/1992 x| BW-CHILD
```

Here is what[ the cataloging rules](https://consul.stanford.edu/display/MD/Bound+withs) have to say about SEE-OTHER:

> For bound-with items, the Symphony child record's Home and Current Locations should be the same as the parent record's permanent Home Location. Child records should NEVER have location INPROCESS.
EXCEPTION: When the bound-with item is in SAL3, both the Home and Current Locations on the child records should always be SEE-OTHER.
In SearchWorks, SEE-OTHER is labeled "See related record(s) to request item" which alerts the user to the fact that there is a related record, and that the related record needs to be retrieved in order to submit a page request for the item.

To sum up I have the following questions:
- For Bound-withs that **are not** SEE-OTHER, is the current request UX acceptable? 
- For Bound-withs that **are** SEE-OTHER, what is the desired request UX, and should it be the same as non-SEE-OTHER Bound-withs?
- Will SEE-OTHER continue to be represented in FOLIO?

**Comment by cbeer** [comment]
> For Bound-withs that are not SEE-OTHER, is the current request UX acceptable?

I'm surprised there are bound-withs with actual item records (and, looking in FOLIO, it looks like that approach is going away).

The UX for [8594988](https://searchworks.stanford.edu/view/8594988) seems fine, although would benefit from general UX improvements to bound-withs (e.g. making sure the user knows what item they'll receive).

> For Bound-withs that are SEE-OTHER, what is the desired request UX

That's the core question #3099 is asking us to investigate.

> Will SEE-OTHER continue to be represented in FOLIO?

No, bound-with child records don't have items in FOLIO.

**Comment by saseestone** [comment]
Confirmation from Jeanette that SEE-OTHER will *not* be used for Bound-with

**Comment by marlo-longley** [comment]
Is it possible to help me find examples of records that exist in BOTH folio-test and current prod SearchWorks that are:

- Bound-with, **no** SEE-OTHER in a note
- Bound-with ,**with** SEE-OTHER in a note

Then I test indexing these FOLIO records. We could use the above examples in this ticket, but they don't yet exist in FOLIO, or I was having trouble finding some.

**Comment by ahafele** [comment]
Hi Marlo, Is this still needed? I'm not sure what you mean by SEE-OTHER in a note - where are you looking to find SEE-OTHER? 
[a2320068](https://folio-test.stanford.edu/inventory/view/b17f10fb-f9ac-5f45-bbb2-ed9e6c4874f5) exists in folio-test but I can provide more examples as needed.

**Comment by marlo-longley** [comment]
These questions were investigated and solved via https://github.com/sul-dlss/searchworks_traject_indexer/pull/894/files
